compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -2164,7 +2164,12 @@ public void visit(BLangFunction funcNode) {
                                                                    getOrigin(funcNode.name.value));
         funcSymbol.source = funcNode.pos.lineRange().filePath();
         funcSymbol.markdownDocumentation = getMarkdownDocAttachment(funcNode.markdownDocumentationAttachment);
-        SymbolEnv invokableEnv = SymbolEnv.createFunctionEnv(funcNode, funcSymbol.scope, env);
+        SymbolEnv invokableEnv;
+        if (env.node.getKind() == NodeKind.CLASS_DEFN || env.node.getKind() == NodeKind.OBJECT_TYPE) {
```

Comment:
We can extract `env.node.getKind()` into separate variable

Reply:
Changed